scripts/install-rkt.sh

```diff
@@ -63,3 +63,6 @@ systemd-tmpfiles --create /usr/lib/tmpfiles.d/rkt.conf
 EOF
 
 checkinstall -y --pkgname=rkt --pkgversion="${version}" ./install-pak
+
+usermod -a -G rkt-admin ubuntu
+usermod -a -G rkt ubuntu
```

Comment:
Sure, but not every user has an `ubuntu` account. This only applies to vagrant installs. `install-rkt.sh` is supposed to work also on a vanilla Ubuntu/Debian to get the latest rkt.